vendor/assets/javascripts/spree/backend/all.js

```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into including all the files listed below.
```

Comment:
These files aren't needed they are automatically added by generators

Reply:
For some reason these files are not added by generators and [specs failed](https://app.travis-ci.com/github/spree/spree_gateway/jobs/534731005#L5715
).
Should I do any change in the configuration?